lib/src/commands/exit.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:args/command_runner.dart';
+import 'package:vscout_cli/src/view/view.dart';
+import 'dart:io';
+
+class ExitCommand extends Command {
+  @override
+  String get name => 'exit';
+  CliView cliView;
+  @override
+  String get description => 'Exit Vscout Command Line Interface';
```

Comment:
vscout CLI  (command line interface is too long, and people recognize CLI over command line interface most of the time)